app/assets/v2/css/kudos/styles.css

```diff
@@ -615,9 +615,9 @@ a.cellarius-partner:before {
   outline: none;
   background-color: transparent;
   display: flex; }
-.flip-card:focus .flip-card__flipper {
+/* .flip-card:focus .flip-card__flipper {
```

Comment:
Is this the wrong flip-card class? Two reasons I'm thinking this:

1. It's located in the `kudos` folder, as opposed to the `grants` folder
2. When on the grants listing page (http://0.0.0.0:8000/grants/?network=rinkeby&category=), clicking the "Add to Cart" button adds the grant to the cart and leaves the card flipped, as shown below. Can we get this to not flip at all when clicking Add to Cart?

![image](https://user-images.githubusercontent.com/17163988/83933279-f0adb580-a75b-11ea-9616-b9d9b5b9f3ca.png)

Reply:
So this is weird, I think if you do a hard refresh, you'll see it doesn't refresh, but I think you're right that this CSS might be used for kudos as well. Uncommented the classes and simply removed them from the grant details div.